index.php

```diff
@@ -1,50 +1,90 @@
 <?php
+require_once 'utilities/DB.php';
 
-function drawCats(array $cats) :string {
+$db = new DB();
+$dbconnection = $db->dbConnect();
+$breeds = [];
+$breedSql = $dbconnection->prepare('SELECT `breed` FROM `breed`');
+$breedSql->execute();
+$breeds = $breedSql->fetchAll();
+
+/***
+ * Iterates through all breeds and populates
+ * the dropdown options
+ *
+ * @param $breeds
+ * @return string
+ */
+
+function populateDropdown(array $breeds): string
+{
+    $stringyBreeds = '';
+
+    for ($i = 0; $i < count($breeds); $i++) {
+        $breed = $breeds[$i]['breed'];
+        $stringyBreeds .= "<option>$breed</option>";
+    }
+    return $stringyBreeds;
+}
+
+/***
+ * Iterates through cat images and displays them on the page
+ *
+ * @param array $cats
+ * @return string
+ */
+
+function drawCats(array $cats): string
+{
     $stringyCats = '';
-    foreach($cats as $cat) {
+    foreach ($cats as $cat) {
         $stringyCats .= '<div class="cat-image"><img src="' . $cat->image . '" alt="A cat"></div>';
     }
     return $stringyCats;
 }
 
+$dropdownBreeds = populateDropdown($breeds);
+
```

Comment:
when I run it, it complains:
Undefined variable: cats in /vagrant/html/2019-top-cat/index.php on line 48

Reply:
yeah mine doesn't run at all